src/python_bindings/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+cmake_minimum_required(VERSION 3.25)
```

Comment:
do we need to change also in RediSearch?

Reply:
not used by RediSearch